src/containers/AddProjectDetails/AddProjectDetailsForm.tsx

```diff
@@ -29,13 +40,113 @@ export default function AddProjectDetailsForm() {
                     }
                 />
                 <InputText
-                    label="Lorem Ipsum"
-                    autoFocus
-                    {...register("email")}
-                    error={!!errors?.email}
-                    helperText={errors.email ? errors?.email.message : null}
+                    label={UIConstants.projectCode}
+                    {...register("projectCode")}
+                    error={!!errors?.projectCode}
+                    helperText={
+                        errors.projectCode ? errors?.projectCode.message : null
+                    }
+                />
+            </styledComp.flexRow>
+            <styledComp.flexRow>
```

Comment:
Can we import the Styled components similar to 
import { **FlexRow**, **FlexAttribute** } from "./AddProjectDetailsForm.style";

Note: Components name should start with the Capital letter.

Reply:
All right, but I have a doubt. will not be something like we'll be importing more when the number of CSS will be higher? i.e.:

import { CSS1, CSS2, CSS3, CSS4, CSS5, CSS6, CSS7, CSS8, CSS9, CSS10} from "./Something.style";

Pls suggest.

Reply:
Yes, we will have more named imports when the styles increases. However, we are also splitting the larger components into smaller pieces by extracting them into a separate component file. Also, we are trying to maintain consistency.

Reply:
okay

Reply:
done